### PR TITLE
Fix #2863: Added Config.yml for Issue Template to disallow blank issues

### LINKED ISSUE #2863: Introduce Configuration File for Issue Template
**Is your feature request related to a problem? Please describe.**
GitHub by default allows having a blanked issue.

**Describe the solution you'd like**
We need to prevent the blank issue from filing.

`.github/ISSUE_TEMPLATE/config.yml`
```yml
# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
blank_issues_enabled: false  # default: true
```

### PR DESCRIPTION
**Pull request by justdvnsh**
## Explanation
This PR creates a config file for the ISSUE TEMPLATE so that creating blank issues can be avoided. Fixes #2863 

## Checklist
- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by anandwana001** [comment]
@justdvnsh Please follow the checklist.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```
Comment (anandwana001) [inline .github/ISSUE_TEMPLATE/config.yml]: Follow as per given in issue. Adding reference and comment gives more context. 

```
# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
blank_issues_enabled: false  # default: true
```

**Review by anandwana001 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (justdvnsh) [reply .github/ISSUE_TEMPLATE/config.yml]: sure. sorry I missed the part.

Comment (justdvnsh) [reply .github/ISSUE_TEMPLATE/config.yml]: @anandwana001 Done.

**Comment by anandwana001** [comment]
@FareesHussain can we test this on fork?

**Comment by FareesHussain** [comment]
> @FareesHussain can we test this on fork?

@justdvnsh From what I've noticed forks don't have an issues tab, you can follow https://softwareengineering.stackexchange.com/a/179470 to make issues available in your fork. 

And to test this I think you need to make your current branch `feat/config_for_issues` a default branch.

**Comment by justdvnsh** [comment]
@FareesHussain I have made the required changes and tested it.  Here is a link to the fork.
https://github.com/justdvnsh/oppia-android

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks
For reference
image
image

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @justdvnsh 
and Thanks @FareesHussain for testing it.

**Review by anandwana001 — CHANGES_REQUESTED** [review]
@justdvnsh Please correct the PR title  The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)

**Comment by anandwana001** [comment]
@justdvnsh 

`Fix #2863: Added Config.yml for Issue Template to disallow blank issues`

**Comment by justdvnsh** [comment]
@anandwana001 Sorry. Missed that.

**Comment by anandwana001** [comment]
Looks like it not fixed now, please take a look.

**Comment by justdvnsh** [comment]
@anandwana001 I am not able to figure out. Sorry. Can you point me the issue ?

**Comment by anandwana001** [comment]
@justdvnsh  `:`

`Fix #2863: Added Config.yml for Issue Template to disallow blank issues`

**Comment by justdvnsh** [comment]
@anandwana001 Is it good to go now ?

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks for the contribution @justdvnsh! Apologies for the noise, but I actually don't think this is something that we want to implement. Will continue discussing it on #2863 before deciding whether to close this pull request.

In the meantime, I suggest looking for one of our other starter issues if you're interested in contributing other changes to the codebase.

**Comment by anandwana001** [comment]
Closing this PR as we had a discussion over team meeting on not to prevent blank issue template.

**Pull request closed without merging**